src/index.css

```diff
@@ -11,3 +11,133 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+/* -------------------- */
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+
+/* ----  HEADER  ---- */
+
+.header{
+  background: #242323;
+  height: 100px;
+  /* border: 2px solid red; */
+}
+
+.header-container{
+  max-width: 1170px;
+  height: 100%;
+  margin: 0 auto;
+
+  display:flex;
+  justify-content: space-between;
+  align-items: center;
+  /* border: 2px solid yellow; */
```

Comment:
Czy te zakomentowane style trafiły tutaj celowo?

Reply:
Tak,  pomogło mi to w weryfikacji układu strony i sprawdzeniu czy wszystkie elementy są rozłożone na stronie tak jak bym chciała. Później usunę te linijki.